src/rise-twitter.js

```diff
@@ -125,7 +125,7 @@ export default class RiseTwitter extends HTMLElement {
     }
 
     if (!this.screenName) {
-      this.logger.error('Error: screenName is missing');
+      this.logger.evt('Warning','screenName is missing');
```

Comment:
@jungeunyoon event types are usually reported in lowercase elsewhere, wouldn't it be good to follow the same convention here ( also, other projects report 'warning' as event type ).

Reply:
@santiagonoguez With Twitter component we moved to capitalizing. Don't think we should take this PR to change all of them but I wouldn't be against changing it for these two cases. Thoughts?